{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/base/utils/routers.py

```diff
@@ -0,0 +1,16 @@
+# Third Party Stuff
+from rest_framework import routers
+
+
+class BaseNameRouter(routers.SimpleRouter):
+    """
+    A router for read-only APIs, which doesn't use trailing slashes.
+    """
+    routes = [
+        routers.Route(
+            url=r'^{prefix}$',
+            mapping={'get': 'list', 'patch': 'partial_update', 'put': 'update'},
+            name='{basename}',
+            initkwargs={'suffix': 'List'}
+        )
+    ]
```

Comment:
let me see, if there can be alternative, if not I would probably use something like this, instead.

```
class SingletonRouter(routers.SimpleRouter):
    """Same as default router but without detail route and GET, POST, PUT, PATCH and 
    DELETE maps to same url as list.
    """
    routes = [
        # List route.
        routers.Route(
            url=r'^{prefix}{trailing_slash}$',
            mapping={
                'get': 'list',
                'post': 'create',
                'patch': 'partial_update',
                'put': 'update',
                'delete': 'destroy',
            },
            name='{basename}',
            initkwargs={'suffix': ''}
        ),
        # Dynamically generated list routes.
        # Generated using @list_route decorator
        # on methods of the viewset.
        routers.DynamicListRoute(
            url=r'^{prefix}/{methodname}{trailing_slash}$',
            name='{basename}-{methodnamehyphen}',
            initkwargs={}
        ),
    ]
```